packages/views/src/view.tsx

```diff
@@ -1,81 +1,49 @@
-import { isActivatable, isDeactivatable } from "@frui.ts/screens";
 import * as React from "react";
+import ErrorBoundary, { ErrorBoundaryProps } from "./errorBoundary";
+import useScreenLifecycle from "./useScreenLifecycle";
 import { getView, tryGetView } from "./viewLocator";
 
 interface ViewProps {
   vm: any;
   context?: string;
   useLifecycle?: boolean;
-  fallbackMode?: "message" | "children";
-  onError?: (error: Error, errorInfo: React.ErrorInfo) => void;
 }
 
-interface ViewState {
-  hasError: boolean;
-}
-
-export default class View extends React.PureComponent<ViewProps, ViewState> {
-  static defaultProps: Partial<ViewProps> = {
-    fallbackMode: "children",
-  };
-
-  static getDerivedStateFromError(error: any) {
-    return { hasError: true };
-  }
+const PureView: React.FunctionComponent<ViewProps> = props => {
+  const { vm, children, context, useLifecycle } = props;
 
-  componentDidCatch(error: Error, errorInfo: React.ErrorInfo) {
-    this.props.onError?.(error, errorInfo);
+  if (!vm) {
+    return null;
```

Comment:
What if we return `children` here as well? Would it break anything? We could then use an empty placeholder as `children`, instead of `vm ? <View ... /> : <EmptyPlaceholder />`.

Reply:
I think that null is on right place here. 

```<View vm={vm.activeChild} />```
Typically you want to render nothing when activeChild is not presented.